src/ray/gcs/gcs_server/test/gcs_placement_group_scheduler_test.cc

```diff
@@ -68,6 +68,16 @@ class GcsPlacementGroupSchedulerTest : public ::testing::Test {
     EXPECT_TRUE(WaitForCondition(condition, timeout_ms_.count()));
   }
 
+  std::vector<std::shared_ptr<gcs::GcsPlacementGroup>> &GetSuccessPlacementGroups() {
```

Comment:
This is not really safe right? This vector still can be modified?

Reply:
yes, I have delete it,  I have made a specialized method for ut check where this two vector used, maybe you can review this  :)